check_process

```diff
@@ -16,16 +16,10 @@
 		setup_private=1
 		setup_public=1
 		upgrade=1
-		# 4.9~ynh1
```

Comment:
I don't like the idea of removing an upgrade test, just to pass tests

Reply:
Me neither: it's because said test couldn't install anymore :)